examples/package.json

```diff
@@ -32,6 +32,8 @@
     "webpack": "^1.13.1"
   },
   "dependencies": {
+    "monaco-editor": "^0.9.0",
```

Comment:
IMHO yes, like in this case https://github.com/reactjs/redux/blob/master/examples/real-world/package.json